operator/controllers/cleanup.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	flowv1beta1 "github.com/banzaicloud/logging-operator/pkg/sdk/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func (r *FlowTestReconciler) cleanUpResources(ctx context.Context, flowTestName string) error {
+	logger := log.FromContext(ctx)
+
+	matchingLabels := &client.MatchingLabels{"loggingplumber.isala.me/flowtest": flowTestName}
```

Comment:
we should pull this into a function in case we ever change our label

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/14

Reply:
This will get addressed in https://github.com/MrSupiri/rancher-logging-explorer/issues/16